openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -515,6 +516,17 @@ def _inner(request):  # pylint: disable=missing-docstring
         else:
             response.content = msg
 
+        # Fire an event for login attempt
+        event_name = 'edx.user.attempt_login'
```

Comment:
I believe `edx.user.login` is a more appropriate name.

Reply:
Same event will be generated either login attempt is successful or not. Therefore I think this name is suitable ? What do you think ?

Reply:
changed.